libs/data-mapper/src/lib/utils/Function.Utils.ts

```diff
@@ -134,3 +135,25 @@ export const functionsForLocation = (functions: FunctionDictionary, targetKey: s
   Object.fromEntries(
     Object.entries(functions).filter(([_key, value]) => value.functionLocations.some((location) => location.key === targetKey))
   );
+
+export const getFunctionLocationsForAllFunctions = (
+  dataMapConnections: ConnectionDictionary,
+  flattenedTargetSchema: SchemaNodeDictionary
+): FunctionDictionary => {
+  const functionNodes: FunctionDictionary = {};
```

Comment:
Nit: add a little white space to this to break it into logical sections?

Reply:
done